pkg/reporter/reporter.go

```diff
@@ -60,11 +61,11 @@ func (r *Reporter) PrintText(msg string) {
 func (r *Reporter) PrintResult(vulnResult *models.VulnerabilityResults) error {
 	switch r.format {
 	case "json":
-		return PrintJSONResults(vulnResult, r.stdout)
+		return output.PrintJSONResults(vulnResult, r.stdout)
 	case "markdown":
-		PrintMarkdownTableResults(vulnResult, r.stdout)
+		output.PrintMarkdownTableResults(vulnResult, r.stdout)
 	case "table":
-		PrintTableResults(vulnResult, r.stdout)
+		output.PrintTableResults(vulnResult, r.stdout)
 	}
```

Comment:
nit:
Since it's public, I think it'd make sense to make `Reporter` an interface, with individual json/markdown/table/void reporter implementations.

That way, an external program could write/wrap their own logging/formatting outputs.

(Also, it avoids this awkward format checking)

Reply:
This sounds like a interesting idea, I'll make the changes and push it in a separate PR.